.gitignore

```diff
@@ -16,3 +16,4 @@ edgesite/edgesite
 keadm/keadm
 _output
 _tmp
+build/edge/tmp
```

Comment:
which generates it?

Reply:
[Makefile#L278](https://github.com/kubeedge/kubeedge/blob/2f4447c60b767de371a8b773774662711ead3089/Makefile#L278)`edgeimage`